test/external-dependencies/boost.cpp

```diff
@@ -22,12 +22,14 @@
 
 #include <boost/test/unit_test.hpp>
 #include <libdevcore/Common.h>
+#include <test/TestHelper.h>
 
 BOOST_AUTO_TEST_SUITE(ExtDepBoost)
 
 // test that reproduces issue https://github.com/ethereum/cpp-ethereum/issues/1977
 BOOST_AUTO_TEST_CASE(u256_overflow_test)
 {
+	dev::test::TestOutputHelper::initTest();
```

Comment:
This line replaces  cnote<<testCaseName;
is some BOOST cases. such as BlockChain, Vm, State,  Transaction  
we use a different method to output the test case name.
thus I have TestOutputHelper::initTest(json::mObject);
and TestOutputHelper::initTest();

I am not quite sure that boost allows to add a user defined function to be called before each boost test case function